src/tests/test_integration/test_shopping_cart.py

```
@@ -15,6 +15,7 @@
 from sqlalchemy.ext.asyncio import AsyncSession
 from tests.conftest import client, db_session, seed_user_groups
```

Ruff (0.11.9)

`tests.conftest.client` imported but unused

Remove unused import

(F401)